src/source/kafka.rs

```diff
@@ -152,8 +152,8 @@ impl rentals::MessageStream {
     // ALLOW: https://github.com/tremor-rs/tremor-runtime/issues/1023
     #[allow(mutable_transmutes, clippy::mut_from_ref)]
     unsafe fn mut_suffix(
-        &self,
-    ) -> &mut stream_consumer::MessageStream<'static, LoggingConsumerContext, SmolRuntime> {
+        &mut self,
+    ) -> &mut stream_consumer::MessageStream<'_, LoggingConsumerContext, SmolRuntime> {
```

Comment:
:sob:

Reply:
pls don't cry. This change is for the better!

Reply:
I know, but I'll be crying and sobbing in the corner until we finally can get rid of it, each transmute is a tear for me

Reply:
Each unsafe transmute kills a little kitten when executed.

Reply:
:sob: